tests/builtins/test_enumerate.py

```diff
@@ -2,7 +2,55 @@
 
 
 class EnumerateTests(TranspileTestCase):
-    pass
+    def test_enumerate(self):
```

Comment:
@cflee Added test for empty iterable. Regarding lazy iterable, I talked to @eliasdorneles regarding this issue, he asked me to implement the list version for the time being. I see this PR has been marked "not-quite-right". could you suggest changes to make it fit for merge?